### ISSUE TITLE
Error when running mesmer on multi-channel image

### ISSUE THREAD

**Issue by apal6** [vanvalenlab/deepcell-applications#40]
Hi,

I have been working on image segmentation using mesmer through singularity image. I was able to run the pipeline on DAPI tif file but when I used the multi-channel image, it gave an error:

` underlay of /usr/bin/nvidia-cuda-mps-control required more than 50 (512) bind mounts
2024-02-02 11:58:36.730947: I tensorflow/core/platform/cpu_feature_guard.cc:151] This TensorFlow binary is optimized with oneAPI Deep Neural Network Library (oneDN
N) to use the following CPU instructions in performance-critical operations:  AVX2 FMA
To enable them in other operations, rebuild TensorFlow with the appropriate compiler flags.
2024-02-02 11:58:38.607682: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1525] Created device /job:localhost/replica:0/task:0/device:GPU:0 with 9653 MB memor
y:  -> device: 0, name: NVIDIA GeForce RTX 2080 Ti, pci bus id: 0000:89:00.0, compute capability: 7.5
WARNING:tensorflow:No training configuration found in save file, so the model was *not* compiled. Compile it manually.
[2024-02-02 11:58:45,363]:[WARNING]:[tensorflow]: No training configuration found in save file, so the model was *not* compiled. Compile it manually.
[2024-02-02 12:02:22,355]:[WARNING]:[root]: Found constant value array in batch 0 and channel 1. Normalizing as zeros.
2024-02-02 12:02:37.277937: I tensorflow/stream_executor/cuda/cuda_dnn.cc:368] Loaded cuDNN version 8100
/usr/local/lib/python3.8/dist-packages/deepcell_toolbox/deep_watershed.py:108: UserWarning: h_maxima peak finding algorithm was selected, but the provided image is
 larger than 5k x 5k pixels.This will lead to slow prediction performance.
  warnings.warn('h_maxima peak finding algorithm was selected, '
/usr/local/lib/python3.8/dist-packages/deepcell_toolbox/deep_watershed.py:179: FutureWarning: `selem` is a deprecated argument name for `h_maxima`. It will be remo
ved in version 1.0. Please use `footprint` instead.
  markers = h_maxima(image=maxima,
Segmentation fault`

How could I fix this? 
Thank you.

**Comment by rossbar** [comment]
Check out the [README](https://github.com/vanvalenlab/deepcell-applications/?tab=readme-ov-file#script-arguments) for details on how to handle multi-channel images; i.e. how to specify nuclear and/or membrane channels for whole-cell segmentation.
